src/main/java/com/sfuentes/census/ResultCensus.java

```diff
@@ -0,0 +1,19 @@
+package com.sfuentes.census;
+
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@RequiredArgsConstructor
+@Getter
+public class ResultCensus {
+  private final List<Integer> censusNumberCount = new ArrayList<>();
+
+  public void addCensusNumberSingleWomenBetweenSixteenAndTwentyOne(Card card) {
+    if (card.getGender().equals("female") && card.getAge() >= 16 && card.getAge() <= 21
+        && card.getCivilStatus().equals("single"))
+      censusNumberCount.add(card.getCensusNumber());
```

Comment:
Why to add the Id to the list?

Reply:
Because, it was the only way I thought about how to print the census number.